pkgs/development/python-modules/luddite/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, pytestCheckHook
+, pytest-cov
+, pytest-socket
+, pytest-mock
+}:
+
+buildPythonPackage rec {
+  pname = "luddite";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "jumptrading";
+    repo = pname;
+    # Requires pre-release patch from master branch
+    rev = "e26c0f90629fd362f3683477c0b7790899f43779";
+    sha256 = "IgpK4N+zPjRa8xOC4JfWEOOZ62aQGbUaPVc2cFfL0fU=";
+  };
+
+  propagatedBuildInputs = [ setuptools ];
+
+  checkInputs = [ pytestCheckHook pytest-cov pytest-socket pytest-mock ];
+
```

An additional safeguard

```suggestion
  pythonImportsCheck = [ "luddite" ];

```